### DISCUSSION TITLE
Installation via conda with numpy>1.21

### DISCUSSION THREAD

**Discussion by ikrommyd** [scikit-hep/coffea#748] [Q&A] [answered]
The conda-forge package requires numpy >=1.16.0,<1.22 for versions newer than 0.7.16 forcing me to install 0.7.15 if I want to use it with a newer numpy version.
What is the reason for this? Is it intentional?
I would assume that newer versions of numpy wouldn't have compatiability issues now that they are all compatiable with numba.

**Comment by lgray** [comment, marked as answer]
I'll unpin it for the latest release, we had to do that funny pinning because of python-version support mixing with available numpy and numba. Gave pip a hard time, but it is sorted now.

**Comment by ikrommyd** [comment]
Yeah I was looking at the requirements and couldn't find a reason for this. I installed all the dependencies via conda and only coffea via pip to get the most recent version.
For some reason, If I did `pip install coffea` within my conda env and let pip handle the dependencies, correctionlib compilation would fail.

**Reply by lgray** [reply]
Huh - correctionlib *shouldn't* be compiling for common architecture/os combinations. What are you running?

**Reply by ikrommyd** [reply]
I know, which is why it is weird. I'm on macOS 12.6.1 in a Macbook M1 Pro.
For some reason in any other conda environment that I have and in system wide python, correctionlib would install normally via pip.
In my conda environment that I have for ROOT, it would not.
Here is a dropbox link of the environment.yml file: https://www.dropbox.com/s/obby4ok7j50j7ur/environment.yml?dl=0

Anyways I have everything from conda now that you have unpinned the numpy version limit. Thank you.

**Reply by lgray** [reply]
Oh - I would highly recommend installing root through brew and avoiding the root conda setups. I have had an unpleasant time with them.

**Comment by lgray** [comment]
https://github.com/conda-forge/coffea-feedstock/pull/35
